doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc

```diff
@@ -0,0 +1,28 @@
+[id='configuring-emails-in-task-notification-proc']
+
+= Configuring emails in task notification
+
+Earlier it was possible to send notifications only to users or group of users in {CENTRAL}. Now you can directly add any email addresses as well.
+
+.Prerequisites
+
+You have created a project in {CENTRAL} and it contains at least one business process asset.
+
+.Procedure
+
+. Create a business process.
+. In the process designer, select the *Activities* tool from the tool palette.
+. Select *User*.
+. Drag and drop a user task onto the process designer canvas.
```

Comment:
I clubbed there steps and converted into one step as "**Create a user task**". 
I have also added the cross-reference links for more details about creating a user task. 

I applied the same method for the step "**Create a business process.**" 
This will explain and clarify more. 
Can you please review the same? 
Thank you.